README.md

```diff
@@ -3,6 +3,7 @@
 - 🐬 [Senior Blockchain Developer](/sr-dev.md)
 - 🐙 [Application Engineer - Enterprise Team](/ae.md)
 - 🦈 [Senior Blockchain Developer - Data Farming](/df.md)
+- 🐳 [Senior Web Developer](/web.md)
```

Comment:
Datapoints:
- "Senior blockchain developer" was written to be general enough for both FE or BE devs. 
- devs like "blockchain" in the title (or "web3", "smart contract", ..). Especially if blockchain is the major part of their work.

It's ok to have more specialized roles, like the proposal here. Really fine. Then also let's:
- replace "Senior Blockchain Developer" with "Senior Backend Blockchain Developer"
- and change "Senior Web Developer" title to have something blockchain. Ideas: "Senior Frontend Blockchain Developer", "Senior Frontend Web3 Developer",

Reply:
In my perspective, the Senior Blockchain Developer is more like a FullStack engineer and not just backend. 
I added this specific Frontend Job description for the person we are searching for to build UI on our market, but blockchain(web3) experience is still a requirement

Reply:
Yes, that's fine. But you need a title more Web3-y than "Senior Web Developer", that's all:)